### PR TITLE
added test report coverage

### PR DESCRIPTION
**Pull request by galmail**
This PR includes test reports with coverage, which is generated as an html inside the **test-report** folder

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -14,5 +14,21 @@ module.exports = {
     '/tests/e2e/',
     '/build/',
   ],
-  reporters: ['default', 'jest-junit'],
+  coverage: true,
```
Comment (jensdev) [inline jest.config.js:17]: I think this should be `collectCoverage`

File: package.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
     "add-suffix": "node ./node_modules/npm-version-suffix/run-add-suffix.js",
     "remove-suffix": "node ./node_modules/npm-version-suffix/run-remove-suffix.js",
     "prepare": "yarn build && husky install",
-    "test": "jest",
+    "test": "jest --coverage",
```
Comment (jensdev) [inline package.json:25]: This is not needed

File: jest.config.js (hunk 1 above)
Comment (galmail) [reply jest.config.js]: oh yes you're right!! I changed it now!

File: package.json (hunk 2 above)
Comment (galmail) [reply package.json]: ah yes! with the flag set in the jest config, this is not needed. I've removed it now.

**Review by jensdev — APPROVED** [review]

**Pull request merged**